tests.py

```diff
@@ -11,8 +11,9 @@
 from django.http import HttpResponse
 
 import request_logging
-from request_logging.middleware import LoggingMiddleware, DEFAULT_LOG_LEVEL, DEFAULT_COLORIZE, DEFAULT_MAX_BODY_LENGTH,\
-    NO_LOGGING_MSG
+from request_logging.middleware import \
```

Comment:
Can you add a quick test to prove the expected behavior with Unicode content? Looks like there are also some conflicts to merge.

Reply:
Added them in 86d35c, but whoops :(
I'll come back after figuring out what's wrong